src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+    }
+
+    /**
+    * Hide the confirmation modal
+    */
+    hideConfirmModal() {
+        this.setState(prevState => ({...prevState, isRemoveMembersConfirmModalVisible: false}));
+    }
+
+    /**
+    * Add or remove user from the selectedEmployees list
+    *
+    * @param {Sting} login
+    */
+    toggleUser(login) {
+        if (login === 'ALL') {
+            this.setState(prevState => ({
+                ...prevState,
+                selectedEmployees: prevState.selectedEmployees.length !== this.props.policy.employeeList.length
+                    ? this.props.policy.employeeList
+                    : [],
+            }));
+            return;
+        }
+
+        this.setState(prevState => ({
+            ...prevState,
+            selectedEmployees: prevState.selectedEmployees.includes(login)
+                ? prevState.selectedEmployees.filter(item => item !== login)
+                : [...prevState.selectedEmployees, login],
+        }));
+    }
+
+    /**
+     * Do not move this or make it an anonymous function it is a method
+     * so it will not be recreated each time we render an item
+     *
+     * See: https://reactnative.dev/docs/optimizing-flatlist-configuration#avoid-anonymous-function-on-renderitem
+     *
+     * @param {Object} args
+     * @param {Object} args.item
+     * @param {Number} args.index
+     *
+     * @returns {React.Component}
+     */
+    renderItem({
+        item,
+    }) {
+        return (
+            <View style={[styles.peopleRow]}>
+                <View style={[styles.peopleRowCell, styles.peopleCheckbox]}>
+                    <Checkbox
+                        isChecked={this.state.selectedEmployees.includes(item.login)}
+                        onPress={() => this.toggleUser(item.login)}
+                    />
+                </View>
+                {
+                    this.props.isSmallScreenWidth ? (
+                        <View style={[styles.peopleRowCell, styles.flex4]}>
+                            <View style={[styles.avatarWithName]}>
+                                <Avatar
+                                    imageStyles={[styles.mr2]}
+                                    source={item.avatar}
+                                />
+                                <View style={[styles.dFlex, styles.flexColumn, styles.justifyContentCenter]}>
+                                    <Text style={[styles.textStrong, styles.peopleMobileAssigneeText]}>
+                                        {item.displayName}
+                                    </Text>
+                                    <Text style={[styles.textLabel, styles.colorMuted, styles.peopleMobileAssigneeText]}>
+                                        {item.login}
+                                    </Text>
+                                </View>
+                            </View>
+                        </View>
+                    ) : (
+                        <Fragment>
```

Comment:
NAB/FYI can also use the `<></>` syntax for fragment if we do not need them keyed

Reply:
Perfect, was not sure if it is a best practise here. Gonna use it from now on.